submissions/unabyband/popup-task/index.html

```diff
@@ -0,0 +1,197 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+        <link rel="stylesheet" href="css/main.css">
+        <title>unabyband pop-up</title>
+    </head>
+    <body>
+        <div class="header">
+            <div class="top_menu">
+                <div class="top_menu__item-container">
+                    <a href="#" class="top_menu__item top_menu__item_mail">Mail</a>
+                </div>
+                <div class="top_menu__item-container">
+                    <a href="#" class="top_menu__item top_menu__item_images">Images</a>
+                </div>
+                <div class="top_menu__item-container">
+                    <input class="input_popup" type="checkbox" id="moreoption" name="moreoption">
+                    <label class="button_popup" for="moreoption"></label>
+                    <div class="popup">
+                        <div class="popup_window">
+                        <ul class="popup_window_list popup_window_visible">
+                        <li class="popup_window_list__item">
+                            <a href="#" class="popup_window__active">
+                                <div class="popup_window__img">
```

Comment:
div looks redundant here, you can handle it without wrapping the image

Reply:
Done